javascript/javascript1/week3/notePad.js

```diff
@@ -0,0 +1,50 @@
+const notes = []; // collection of objects
+function saveNote(content, id) {
+  let elem = {}; // first we made an object var
+  elem.id = id;
+  elem.note = content;
+  notes.push(elem); // we pushed the object into arr
+
+  // write some code here
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes);
+
+// here go getNot function
+
+function getNote(id) {
+  for (i = 0; i < notes.length; i++) {
+    if (id === notes[i].id) {
+      return notes[i];
+    }
+  }
+  if (isNaN(id)) {
+    console.log("the input is not a number ");
+  } else {
+    console.log("there's no note with this id ");
+  }
+
+  // your code here
+}
```

Comment:
I recommend throwing an error after for loop like this
```js
  function getNote(id) {
    for (let i = 0; i < notes.length; i++) {
      if (notes[i].id === id) {
        return notes[i];
      }
      throw new Error('note not found!');
    }
  }
```
inside for loop, if you get the note and return it, then the function will stop at `return` and don't continue.
Otherwise, it will reach throw error part